app/modules/intelligence/provider/provider_service.py

```
@@ -29,6 +29,9 @@
 from pydantic_ai.models.openai import OpenAIModel
 from pydantic_ai.models.anthropic import AnthropicModel
 from pydantic_ai.providers.openai import OpenAIProvider
+import litellm
```

Ruff (0.8.2)

Redefinition of unused `litellm` from line 10

(F811)